api_tests/nodes/views/test_node_detail.py

```diff
@@ -1197,6 +1197,21 @@ def make_payload(self, node_id, license_id=None, license_year=None, copyright_ho
     def make_request(self, url, data, auth=None, expect_errors=False):
         return self.app.patch_json_api(url, data, auth=auth, expect_errors=expect_errors)
 
+    def test_admin_update_license_with_invalid_id(self):
```

Comment:
Verified that this and the `preprints` version produced the expected error on `develop`:
```
api/nodes/serializers.py:99: in get_license_details
    license_id = validated_data['license_type'].license_id
E   AttributeError: 'unicode' object has no attribute 'license_id'
```